src/tests/member.test.js

```diff
@@ -1 +1,67 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import Member from '../models/member';
+import memberSeed from '../seeds/member';
+
+/* const mockMember = {
+  firstName: 'Adolf',
+  lastName: 'Rodriguez',
+  dni: 34000981,
+  phone: 5372456337,
+  email: 'rpo23lethwaitei@google.com',
+  password: 'u1Xas6cZ8',
+  city: 'Ciparay',
+  birthDay: '1979-07-12T03:00:00.000Z',
+  postalCode: 86049,
+  isActive: false,
+  membership: 'Black',
+}; */
+
+beforeEach(async () => {
+  await Member.collection.insertMany(memberSeed);
+});
+
+afterEach(async () => {
+  await Member.collection.deleteMany();
+});
+
+describe('GET /api/members', () => {
+  test('Should bring data', async () => {
+    const response = await request(app).get('/api/members').send();
+    expect(response).toBeTruthy();
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+  test('If wrong URL status 404', async () => {
```

Comment:
I don't know if you have to test if the URL is okey, because you are not doing the validation of that and in fact the browser in self is going to return 404 not found

Reply:
in activities i checked the wrong url, and nai approve it, i suppose is not bad to add it.